persistence/module.go

```diff
@@ -237,7 +235,7 @@ func (m *persistenceModule) GetTxIndexer() indexer.TxIndexer {
 	return m.txIndexer
 }
 
-func (m *persistenceModule) GetTreeStore() modules.TreeStore {
+func (m *persistenceModule) GetTreeStore() modules.TreeStoreModule {
```

Comment:
I am confused as this is present but has been removed from the `PersistenceModule` interface?

Reply:
Good catch, this was dead code, it's removed now. 👍